dina-base-api/src/test/java/ca/gc/aafc/dina/security/DinaAdminCUDAuthTest.java

```diff
@@ -33,30 +26,38 @@
 import org.springframework.stereotype.Service;
 import org.springframework.validation.SmartValidator;
 
-import javax.inject.Inject;
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.Id;
-import javax.transaction.Transactional;
-import java.time.OffsetDateTime;
-import java.util.Optional;
-import java.util.UUID;
+import ca.gc.aafc.dina.TestDinaBaseApp;
+import ca.gc.aafc.dina.dto.RelatedEntity;
+import ca.gc.aafc.dina.entity.DinaEntity;
+import ca.gc.aafc.dina.jpa.BaseDAO;
+import ca.gc.aafc.dina.mapper.DinaMapper;
+import ca.gc.aafc.dina.repository.DinaRepository;
+import ca.gc.aafc.dina.repository.DinaRepositoryIT;
+import ca.gc.aafc.dina.service.AuditService;
+import ca.gc.aafc.dina.service.DefaultDinaService;
+import ca.gc.aafc.dina.testsupport.security.WithMockKeycloakUser;
 
-import static org.junit.jupiter.api.Assertions.*;
+import io.crnk.core.queryspec.QuerySpec;
+import io.crnk.core.resource.annotations.JsonApiId;
+import io.crnk.core.resource.annotations.JsonApiResource;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
 
 @Transactional
-@SpringBootTest(classes = {TestDinaBaseApp.class, DinaAdminOnlyAuthTest.DinaAdminOnlyTestConfig.class},
+@SpringBootTest(classes = {TestDinaBaseApp.class, DinaAdminCUDAuthTest.DinaAdminCUDTestConfig.class},
   properties = "keycloak.enabled: true")
-public class DinaAdminOnlyAuthTest {
+public class DinaAdminCUDAuthTest {
```

Comment:
Changes for this file:

* Renamed the test, also renamed the authorization service.
* Imports were cleaned up.
* Added a new read test (with and without access)
* TYPE_NAME is now public like the rest of the TYPE_NAME variables.